vite.browser.config.ts

```diff
@@ -0,0 +1,35 @@
+import esbuild from 'esbuild'
+import { defineConfig, Plugin } from 'vite'
+
+const minifyBundle = (): Plugin => ({
+  name: 'minify-bundle',
+  async generateBundle(_, bundle) {
+    for (const asset of Object.values(bundle)) {
+      if (asset.type == 'chunk')
+        asset.code = (
+          await esbuild.transform(asset.code, {
+            minify: true,
+            legalComments: 'none',
```

Comment:
This file is for WebExtensions, right?
If so, please change to `legalComments: 'eof'`.

Reply:
Can do!